README.md

```diff
@@ -196,6 +196,7 @@ The request from Manticore Search to Buddy is made in JSON format no matter how
 | `error` | An object containg information about error(error message, etc.) to be returned to the user, if any. |
 | `message` | An object containing details such as `path_query` (specific to JSON over HTTP requests), `http_method`  (`HEAD`, `GET`, etc) and `body` which holds the main content of the request. For JSON over HTTP, `path_query` can include specific endpoints like `_doc`, `_create`, etc., while for SQL over HTTP/mysql, it remains empty (`""`). `http_method` is set to `""` for SQL over HTTP/mysql |
 | `version` | The maximum protocol version supported by the sender. |
+| `meta` | A JSON object containing additional information about the response or null. |
```

Comment:
I think it's important to mention the fields important for the daemon (for query logging and `show meta`).